<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OBSAdvAudioCtrl</class>
 <widget class="QWidget" name="OBSAdvAudioCtrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>46</height>
   </rect>
  </property>
  <widget class="QLabel" name="iconLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </widget>
  <widget class="QWidget" name="activeContainer" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>65</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>120</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="activeLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="active">
      <property name="text">
       <string>Active</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QWidget" name="stackedVolume">
    <widget class="QDoubleSpinBox" name="volume">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="suffix">
      <string> dB</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="singleStep">
      <double>0.1</double>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="stackedPercent">
    <widget class="QSpinBox" name="percent">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="forceMonoContainer" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>88</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>50</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="forceMonoLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QCheckBox" name="forceMono">
      <property name="text">
       <string>Mono</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="balanceContainer" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>150</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="balanceLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="labelL">
      <property name="text">
       <string>L</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="BalanceSlider" name="balance" native="true">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelR">
      <property name="text">
       <string>R</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSpinBox" name="syncOffset">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="suffix">
    <string> ms</string>
   </property>
   <property name="minimum">
    <number>-950</number>
   </property>
   <property name="maximum">
    <number>20000</number>
   </property>
  </widget>
  <widget class="QComboBox" name="monitoringType">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="mixerContainer" native="true">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>14</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="mixerLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QCheckBox" name="mixer1">
      <property name="accessibleDescription">
       <string>Basic.Settings.Output.Adv.Audio.Track1</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="mixer2">
      <property name="accessibleDescription">
       <string>Basic.Settings.Output.Adv.Audio.Track2</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="mixer3">
      <property name="accessibleDescription">
       <string>Basic.Settings.Output.Adv.Audio.Track3</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="mixer4">
      <property name="accessibleDescription">
       <string>Basic.Settings.Output.Adv.Audio.Track4</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="mixer5">
      <property name="accessibleDescription">
       <string>Basic.Settings.Output.Adv.Audio.Track5</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="mixer6">
      <property name="accessibleDescription">
       <string>Basic.Settings.Output.Adv.Audio.Track6</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BalanceSlider</class>
   <extends>QSlider</extends>
   <header>balance-slider.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
